pkg/storage/node.go

```diff
@@ -78,8 +84,15 @@ func (s *nodeStorage) Store(newNodes []NodeMetricsPoint) {
 				// Move stored point to previous
 				prevNodes[newNode.Name] = lastNode
 			} else if prevNode, found := s.prev[newNode.Name]; found {
-				// Keep previous point
-				prevNodes[newNode.Name] = prevNode
+				if prevNode.Timestamp.Before(newNode.Timestamp) {
+					// Keep previous point
+					prevNodes[newNode.Name] = prevNode
+				} else {
+					klog.InfoS("Found new node metrics point is older then stored previous, drop previous",
```

Comment:
Let's increase verbosity of this log. Can be done as separate PR
```suggestion
					klog.V(2)InfoS("Found new node metrics point is older then stored previous, drop previous",
```